workers/src/bin/dvn.rs

```diff
@@ -92,7 +94,15 @@ async fn main() -> Result<()> {
                                         dvn_data.verifying();
                                         debug!("Packet NOT verified. Calling verification.");
 
-                                        // FIXME: logic for NFFL verification
+                                        if log.block_number.is_none() {
+                                            error!("Block number is None, can't verify Packet.");
+                                            continue;
+                                        }
+
+                                        if !verifier.verify(log.block_number.unwrap()).await? {
```

Comment:
Maybe better error handling here other than `unwrap`, otherwise it will panic and ggs